phplib/base.php

```diff
@@ -30,10 +33,8 @@
 $sleep_state_levels = array(-1 => "Unknown", 1 => "NREM Stage 1", 2 => "NREM Stage 2", 3 => "NREM Stage 3", 4 => "REM");
 
 
-// Test to make sure we can handle this team
-// .. and handle dev
-$fqdn = (isset($fqdn)) ? $fqdn : $_SERVER['HTTP_HOST'];
-$fqdn = preg_replace($dev_fqdn, $prod_fqdn, $fqdn);
+// Test to make sure we can handle this team.
+$fqdn = (new FQDN(isset($fqdn) ? $fqdn : $_SERVER['HTTP_HOST'], $dev_fqdn, $prod_fqdn))->getFQDN();
```

Comment:
could you unpack this statement a bit? This is pretty hard to parse with the ternary statement in there.